public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx

```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
-            </div>
-            <section className='results'>
-                <h5 className='result-heading secondary-display'> EHR Domains:</h5>
-                <TooltipReactComponent
-                    label='Homepage Tooltip Hover'
-                    searchTerm={this.state.searchWord}
-                    action='Tooltip Home Page EHR Domains'
-                    tooltipKey='ehrDomainHelpText' />
-                <div id='survey' className='result-boxes'>
-                    {
-                        this.state.domainInfo.map((domain, index) => {
-                            const key = 'domain' + index;
-                            return <ResultLinksComponent key={key} {...domain} />;
-
-                        })
-
-                    }
-                </div>
-                <h5 className='result-heading secondary-display'>Survey Questions:</h5>
-                <div className='result-boxes'>
-                    {
-                        this.state.surveyInfo.map((survey, index) => {
-                            const key = 'survey' + index;
-                            return <ResultLinksComponent key={key} {...survey} />;
-                        })
-
-                    }
-                </div>
-                <h5 className='result-heading secondary-display'>
-                    Physical Measurements and Wearables:</h5>
-                <div className='result-boxes'>
-                    {
-                        this.state.physicalMeasurementsInfo.map((phyMeasurements, index) => {
-                            const key = 'phyMeasurements' + index;
-                            return <ResultLinksComponent key={key} {...phyMeasurements} />;
-                        })
-                    }
+            const { domainInfo, physicalMeasurementsInfo, surveyInfo, searchWord, popUp } = this.state;
+            return <React.Fragment>
+                <style>{css}</style>
+                <h1 style={{ ...globalStyles.primaryDisplay, ...styles.dBTitle }}>Data Browser</h1>
+                <p style={{ ...styles.dBDesc, ...globalStyles.bodyLead }}>
+                    The Data Browser provides interactive views of the publicly available<i>All of Us </i>
+                    Research Program participant data. Currently, participant provided information, including surveys and physical
+                    measurements taken at the time of participant enrollment, as well as electronic health record data (EHR) are available.
+                EHR data are reported by health care providers and are not participant reported. The <i>All of Us </i>
+                    Research Program data will include more data types over time.<br></br><br></br>
+                    In order to protect participant privacy, the data are de-identified, limited to aggregate counts rounded up to counts of
+                20, and summary demographic information. For more information, please visit our FAQ page.<br></br><br></br>
```

Comment:
Nit: use self-closing tag:
<br/><br/>
```suggestion
                20, and summary demographic information. For more information, please visit our FAQ page.<br/><br/>
```